driver-async/src/test/functional/com/mongodb/async/client/InitialDnsSeedlistDiscoveryTest.java

```diff
@@ -96,48 +94,35 @@ public void shouldResolve() {
             assertTrue(connectionString.getHosts().containsAll(seeds));
 
             for (Map.Entry<String, BsonValue> entry : options.entrySet()) {
-                if (entry.getKey().equals("connectTimeoutMS")) {
-                    assertEquals(entry.getValue().asNumber().intValue(), (int) connectionString.getConnectTimeout());
-                } else if (entry.getKey().equals("replicaSet")) {
+                if (entry.getKey().equals("replicaSet")) {
                     assertEquals(entry.getValue().asString().getValue(), connectionString.getRequiredReplicaSetName());
-                } else if (entry.getKey().equals("socketTimeoutMS")) {
-                    assertEquals(entry.getValue().asNumber().intValue(), (int) connectionString.getSocketTimeout());
-                } else if (entry.getKey().equals("readPreference")) {
-                    assertEquals(entry.getValue().asString().getValue(), connectionString.getReadPreference().getName());
-                } else if (entry.getKey().equals("readPreferenceTags")) {
-                    assertEquals(asTagSetList(entry.getValue().asArray()),
-                            ((TaggableReadPreference) connectionString.getReadPreference()).getTagSetList());
+                } else if (entry.getKey().equals("ssl")) {
+                    assertEquals(entry.getValue().asBoolean().getValue(), connectionString.getSslEnabled());
+                } else if (entry.getKey().equals("authSource")) {
+                    // ignoring authSource for now, because without at least a userName also in the connection string,
```

Comment:
This is unfortunate but would require more surgery in ConnectionString than I want to take on right now in order to be able to make an assertion here.  So instead testing it on ConnectionStringSpecification

Reply:
Ack